modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -21,7 +21,7 @@
 
 IMPORTANT: Please check both the advanced and expert configurations carefully before using in production.
 
-ownCloud ships with an LDAP application, called user_auth_ldap.
+ownCloud has the option to install a LDAP application called "LDAP Integration". You find it in the marketplace.
```

Comment:
This could be worded better as: "ownCloud can be integrated with an LDAP server via the LDAP Integration app, available in the ownCloud Marketplace.", with a link to the app in the marketplace.

Reply:
I would write both here, the Marketplace Name and the actual app name.